nncf/common/hardware/configs/npu.json

```diff
@@ -382,7 +392,7 @@
             "type": "Embedding",
             "quantization": {
                 "weights": [
-                    "q8_w_sym", "q8_w_asym"
+                    "q8_w_sym", "q8_w_asym", "q8_a", "q8_a_sym", "q8_a_ch"
```

Comment:
Need also verify that NPU supports new schemes.

Reply:
Outdated